kotlin-generator/src/test/scala/models/generator/kotlin/KotlinUtilTest.scala

```diff
@@ -104,6 +104,7 @@ class KotlinUtilTest
   "isModelNameWithPackage" should "return correctly" in {
     isModelNameWithPackage("abc") should be(false)
     isModelNameWithPackage("io.apibuilder.common.v0.models.reference") should be(true)
+    isModelNameWithPackage("io.apibuilder.common.v0.models.testReference") should be (true)
```

Comment:
@mchimirev 
Before the change, `isModelNameWithPackage` would return false and `io.apibuilder.common.v0.models.testReference` would be generated into `IoApiBuilderCommonV0ModelsTestReference` because we didn't catch the package.